Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM php:7.4-apache
+
+WORKDIR /app
+
+ADD docker_files/vhost.conf /etc/apache2/sites-available/000-default.conf
+ADD docker_files/apache.conf /etc/apache2/conf-available/z-app.conf
+ADD docker_files/php.ini /usr/local/etc/php/conf.d/app.ini
+RUN a2enmod rewrite && a2enconf z-app 
+COPY docker_files/errors /errors
+
+COPY index.php /app/index.php
+COPY Cakefile /app/Cakefile
+COPY htaccess.example /app/.htaccess
+RUN sed -i'' -e 's/RewriteBase \/edith\//RewriteBase \//' .htaccess
```

Comment:
I think we might as well just update the example htaccess, actually.

Reply:
As you wish